mep/common/templatetags/mep_tags.py

```diff
@@ -65,3 +69,43 @@ def iiif_image(img, args):
     except (IIIFImageClientException, TypeError):
         # return an empty string if anything goes wrong
         return ''
+
+
+@register.filter
+def ap_format(partial_date):
```

Comment:
This is a good start, made it easier for me to see how to refactor and generalize the way I was thinking of, to make it more like django's `date` filter.

A few notes:
- template tag should explicitly reference partial date
- needed error handling; swallow bad string that can't be parsed as a partial date
- make it clearer in the comments that it's calling the django template tag (otherwise might not be obvious by `date` out of context)